docs/source/array-chunks.rst

```diff
@@ -104,7 +107,7 @@ Unknown chunksizes also occur when using a Dask DataFrame to create a Dask array
    >>> ddf.to_dask_array()
    dask.array<..., shape=(nan, 2), ..., chunksize=(nan, 2)>
 
-Using func:`dask.dataframe.DataFrame.to_dask_array` resolves this issue:
```

Comment:
This should just need a `:` before func.

Reply:
Can you add this?

Reply:
I have, in the line below (https://github.com/dask/dask/pull/5372/files#diff-ff2e01974940287fe3fdda7e71baedd9R110)